src/PostgREST/DbRequestBuilder.hs

```diff
@@ -46,13 +46,13 @@ readRequest maxRows allRels allProcs apiRequest  =
       let target = iTarget apiRequest in
       case target of
```

Comment:
Though I think that ```toSourceRelation``` already discriminates the retType values to only Tables so the detection might still work for any ```pdReturnType``` but I think that is more clear with the pattern match.